### DISCUSSION TITLE
Adressbook entries

### DISCUSSION THREAD

**Discussion by microspace-rl** [rustdesk/rustdesk#6832] [General]
### Bug Description

I run a Rust-Desk server and have purchased the "Business self-hosting plan" license.
Everything works very well so far, but since a few days I can no longer add new entries to my address book. The command is no longer available in the context menu. Even with existing entries in the address book, the entry to remove the entry in the context menu is missing.
I have a total of 150 entries in my address book. 

### How to Reproduce

Unfortunately, I am not aware of any way to reproduce the behavior

### Expected Behavior

it should be possible to delete and add entries in the adressbook again

### Operating system(s) on local side and remote side

Windows 10 - Ubuntu  22.04

### RustDesk Version(s) on local side and remote side

1.2.2  - 1.2.2

### Screenshots

![image](https://github.com/rustdesk/rustdesk/assets/155453875/ff386d75-6de9-4b7e-9960-8469a98c740e)

**Comment by rustdesk** [comment]
Anyway, please try out 1.2.3 and our nightly build first (you do not need to install, just download and run directly). We do have a lot of changes about address book in 1.2.3 and nightly build.

https://github.com/rustdesk/rustdesk/releases/tag/1.2.3
https://github.com/rustdesk/rustdesk/releases/tag/nightly

**Comment by rustdesk** [comment]
If 1.2.3 and nightly still have problem, please try out log out user and log in again on rustdesk.

![image](https://github.com/rustdesk/rustdesk/assets/71636191/c2e2348c-a7b9-4ce7-8e05-4cf1cb8b45aa)

**Comment by 21pages** [comment]
Please also check whether  `Sync with recent sessions` is selected,  deselecting it if it is selected and retry.

![1704205214576](https://github.com/rustdesk/rustdesk/assets/14891774/2d3cbbb8-2159-4a28-95f7-093b58886d27)

**Comment by microspace-rl** [comment]
Could you tell me how to update the server ? with the update-script from (https://raw.githubusercontent.com/techahold/rustdeskinstall/master/update.sh) it hast installed the Version 1.1.9......

**Comment by microspace-rl** [comment]
I have some new details about the error:
1. server and clients are updated to the latest version
2. in the context menu the item "add to address point" is only missing for users with a large number of entries in the address book. For users who only have a few entries, the menu item is available.
3. if you activate the checkboxes in the interface under Recent and select a client, it is possible for all users to add this selected entry to the address book.

Adding entries therefore works, only the menu item in the context menu is not displayed for users with too many entries in total

**Comment by 21pages** [comment]
For cases where deletion fails, please check whether this message appears after deletion.
![rustdesk_0iTgzhyt59](https://github.com/rustdesk/rustdesk/assets/14891774/f1948498-39fc-44f5-ba2c-84f2c6c6b393)

For `Add to address book`  missing in the context menu, please check whether the ID is already in the address book.
